cmd/extensions/Dockerfile

```diff
@@ -19,7 +19,6 @@ RUN apk --update add ca-certificates && \
     adduser -D -u 1000 agones
 
 COPY --chown=agones:agones ./bin/extensions.linux.$ARCH /home/agones/extensions
-COPY --chown=agones:agones ./bin/LICENSES ./bin/dependencies-src.tgz /home/agones/
```

Comment:
Why is this redundant?

Reply:
I probably should not have completely removed the whole line and I am most likely wrong but I was wondering if copying the license to `/home/agones` was redundant if controller was doing the same thing. I kept this as a separate commit just in case. 😅

Reply:
I would revert this. LICENSES and vendored dependencies are an annoying requirement for some licenses and we need it in each image (probably). But glancing around at how this is generated leads me to think we're actually missing a step in this PR - extensions needs to be added to:

https://github.com/googleforgames/agones/blob/e048859c6ff3da0d4b299f915113993aa49c7865/build/build-required-src-dist.sh#L29

Reply:
Oh, I did not know about this. Thank you!